test/cli/scenarios_to_cli/other /scenario_mutate_endpoint/kyverno-test.yaml

```diff
@@ -0,0 +1,12 @@
+name: policy-endpoints
+policies:
+  - policy.yaml
+resources:
+  - resource.yaml
+  patchedresource: test/output/output_mutate_endpoint.yaml
```

Comment:
Didn't catch this earlier. Please place all related files in the same directory. This patchedResource needs to go here. We don't want files scattered everywhere.